### ISSUE TITLE
NTR: Peroxisome enlarged and elongated (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9062]
New term request Peroxisome enlarged and elongated (lyon_2001_03_05_11_27_34):   UMLS: 4314558

**Comment by pnrobinson** [comment]
@MickeySegal

adding 
Adding new term: 
- ID: HP:6000151
- label: Enlarged peroxisomes
- parents: [Abnormal peroxisomal morphology (HP:0034552)](https://hpo.jax.org/app/browse/term/HP:0034552)
- pmid: PMID:2894756
- def: Abnormal increase in size of peroxisomes.
- comment: See figure 2 of PMID:2894756.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9062
- synonyms: Peroxisomal enlargement(NONE)
